pkg/hubble/server/server.go

```diff
@@ -55,18 +55,20 @@ func NewServer(log logrus.FieldLogger, options ...serveroption.Option) (*Server,
 }
 
 func (s *Server) newGRPCServer() (*grpc.Server, error) {
-	switch {
-	case s.opts.Insecure:
-		return grpc.NewServer(), nil
```

Comment:
The commit message doesn't mention anything about this removal. I assume this is fine to be removed because we already check that `ServerTLSConfig` is not nil in the constructor?

Reply:
Yeah, this was always redundant from what I could tell. Constructor handles this and you can't modify listener after creating the Server, so it was never really needed.